docs/noir_js/noir_js.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Noir JS
+description:
+  Learn how to use noir js to use Noir in a Typescript or Javascript environment
+keywords:
+  [
+    Noir project,
+    javascript,
+    typescript,
+    node.js,
+    browser,
+    react
+  ]
+---
+
+Noir JS are a set of typescript libraries that make it easy to use Noir on your dapp, webapp, node.js server, website, etc.
```

Comment:
Is this one package or multiple? If its one, it should say "Noirjs is a set..."

Reply:
Multiple packages for now (@noir-lang/backend_barretenberg and @noir-lang/noir_js). Probably they'll always be two packages, but we're still discussing this